templates/zerver/base.html

```diff
@@ -19,6 +19,9 @@
     <link href="/static/favicon.ico?v=2" rel="shortcut icon">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <link href="/static/node_modules/katex/dist/katex.min.css" rel="stylesheet">
+    <script src="/static/node_modules/katex/dist/katex.min.js"></script>
```

Comment:
Should these be removed now? KaTeX is imported now through `PIPELINE` and `JS_SPECS`, but I don't know if `app.js` and `app.css` are provided as well in development environments.

My guess is that they are, but I'd like to make sure :)